docs/en/advanced_guides/evaluation.md

```diff
@@ -1 +1,56 @@
 # Evaluation
+
+<!-- TOC -->
+
+- [Evaluation](#evaluation)
+  - [Evaluation in MMEngine](#evaluation-in-mmengine)
+  - [Evaluation In MMSelfSup](#evaluation-in-mmselfsup)
+  - [Customize Evaluation](#customize-evaluation)
+
+<!-- /TOC -->
+
+## Evaluation in MMEngine
+
+During model validation and testing, quantitative evaluation is often required. `Metric` and `Evaluator` have been implemented in MMEngine to perform this function. See [MMEngine Doc](https://github.com/open-mmlab/mmengine/blob/main/docs/zh_cn/design/metric_and_evaluator.md).
+
+Model evaluation is divided into online evaluation and offline evaluation, as shown in the following figure:
+
+![img](https://user-images.githubusercontent.com/15977946/163718224-20a4970a-e540-4a3a-8b01-bf0a604c6841.jpg)
```

Comment:
I am a little bit confused by the diagram. Could you please give more detailed explanation?